### ISSUE TITLE
Page List Empty

### ISSUE THREAD

**Issue by 12waffles** [inorichi/tachiyomi#1458]
Version 0.7.4

When I try to open or download a chapter, i get "page list empty". I just got this app today and some of the chapters that get this error were working earlier today.

I'm using ReadComicOnline

**Comment by LakotaGray** [comment]
Have you gone to the extensions section of the menu area and updated to newest readcomicsonline update?

If so, open one of the series you’re getting the issue with, in the browser, make sure there are no captcha, if there are fill it out

**Comment by 12waffles** [comment]
I didn't see this earlier but it seems to have resolved itself. I will use this in the future if it comes up again, though.
Thanks!

**Comment by QuietusWolfe** [comment]
Having this same issue w/ kissmanga. The extension is up to date and there's no capcha as far as I've seen

**Comment by Radixtrator** [comment]
Same thing here with KissManga

**Comment by GummiWummiBear** [comment]
yeah me too i just tried updating to the latest version, clearing cookies and physically visiting the site but no avail

**Comment by xRaTiOnX** [comment]
Still having the same issue, no captcha to fill and page list is empty on all my registered kissmanga series.

**Comment by CarlosEsco** [comment]
@xRaTiOnX   did you update to latest dev or install the kissmanga extension?  If so then try this as kissmanga has cloudflare issues at times.

• In Tachiyomi go to settings advanced  and clear cookies.
• Opened chrome go to website that is causing the issue.
• click a random comic/manga and click a chapter, let it load.
• Open Tachiyomi.
• go to catalouges and click the browse button the source that is having the issue
• it should work, if it does not click the retry button, after a couple retry attempts it should work.

≫ now you can go back to your library and read your chapters/downloads.

**Comment by xRaTiOnX** [comment]
I see, I have the lastest release, also I did not have the kissmanga extension installed prior to having this issue, but I tried downloading the extension now and it prompted to install twice, but with no luck saying that an error ocurred when analysing the package on the second prompt. I also cleared cookies on advanced but still no changes.

I browsed kissmanga and opened a random chapter of a random manga, it loaded fine on google chrome browser, however in tachiyomi when I go into catalogues and try browsing in kissmanga it shows "no results found", even after trying a few dozen times clicking the retry button.

I'm not sure what other information I could provide to help identify my issue, any advice for this?
